functors_task_1/student/include/impl.h

```diff
@@ -16,16 +30,29 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(BeerBrand brand)
 {
+    switch (brand)
+    {
+        case BeerBrand::Corona:
+        case BeerBrand::HoeGaarden:
+        {
+            return true;
```

Comment:
avoid explicit true/false in your code, also just a plain logical expression is a best fit here